jobs/ci-kubernetes-e2e-gce-1.6-1.5-downgrade-cluster.env

```diff
@@ -0,0 +1,14 @@
+### job-env
+
+E2E_OPT=--check_version_skew=false
+
+JENKINS_PUBLISHED_VERSION=ci/latest-1.6
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest-1.5
```

Comment:
Does this mean that we'll use the 1.5 e2e binary to downgrade? Can we use the 1.6 binary so that the tests checked into that branch can be run?